config/routes.php

```diff
@@ -1,4 +1,4 @@
 <?php
-use Cake\Routing\Router;
-
-Router::extensions(['pdf']);
```

Comment:
Instead of remove this line it can be changed to use `Router::addExtensions()` and it will work as intended.